src/Sylius/Component/Core/Model/OrderItem.php

```diff
@@ -57,4 +57,24 @@ public function equals(BaseOrderItemInterface $item)
             && $item->getVariant() === $this->variant
         );
     }
+
+    /**
+     * Returns sum of neutral and non neutral tax adjustments on order item and total tax of units.
+     *
+     * {@inheritdoc}
+     */
+    public function getTaxTotal()
```

Comment:
Maybe we need some kind of `AdjustablesCalculator` service, whereby we pass in a set of adjustables and we can perform calculation methods like this on it. This implementation here will include refunds too, sometimes we want to know what the tax was before refunds so again requiring a different method. Lots of these types of calculation methods will begin to clutter the Adjustable on all three levels. Separating the calculation logic would be better in my opinion; just a thought.